cluster-setup/aws-cluster.yaml

```diff
@@ -153,19 +155,14 @@ head_setup_commands: []
 worker_setup_commands: []
 
 # Command to start ray on the head node. You don't need to change this.
-# ulimit error (non blocker) ray issue #11369
 head_start_ray_commands:
     - ray stop
-    # Ray by default would use all the cpus. We restrict ray to use only as many cpus as the physical cores on the machine.
-    - num_cores="$(grep ^cpu\\scores /proc/cpuinfo | uniq |  awk '{print $4}')";
-      ulimit -n 65536; ray start --head --num-cpus $num_cores --port=6379 --object-manager-port=8076 --autoscaling-config=~/ray_bootstrap_config.yaml
+    - ray start --head --port=6379 --object-manager-port=8076 --autoscaling-config=~/ray_bootstrap_config.yaml
```

Comment:
Do we no longer need to set ulimit?

Reply:
They got rid of it in the template which they ship with 1.6.0